file-formats/srvc/type/zif_aff_srvc_v1.intf.abap

```diff
@@ -0,0 +1,76 @@
+INTERFACE zif_aff_srvc_v1
+  PUBLIC.
+  "! <p class="shorttext">Consumer Type</p>
+  "! Consumer Type
```

Comment:
Use sentence style for all descriptions. Use title style for all "shorttexts". 

This is relevant for all descriptions and "shorttexts"

Reply:
Updated all descriptions